docs/modules/ROOT/pages/references/cp_node_scaling.adoc

```diff
@@ -0,0 +1,51 @@
+= Control Plane Node Scaling
+
+The control plane node resource requirements depent on a number of different metrics.
+CPU requirements generally depend on the ammount on API write requests,
+while memory requirements generally depend on the amount of resource objects (most importantly Pod count) in the cluster.
+
+To avoid cascading failures during a scheduled node reboot or an unexpected node failure,
+it's recommended to keep the overall CPU and memory usage not significantly higher than 60%.
+
+== Guidelines
+
+The following guiedlines are an estimate, based on metrics collected from our managed clusters.
+Individual clusters may vary depending on worker node size and individual workload running on the cluster.
+
+[cols="1,1"]
+|===
+| Control Plane Spec  | Cluster Parameters
+
+| 4 CPU
+
+16Gi Memory
+
+| Small Cluster
+
+max. 3 Worker Nodes
+
+
+| 4 CPU
+
+24Gi Memory
```

Comment:
This size doesn't really make sense on any provider except vSphere where we can do whatever we want. Neither cloudscale nor Exoscale have a 4 CPU / 24Gi VM size.

Reply:
vSphere has 😁